Library/Homebrew/utils.rb

```diff
@@ -770,3 +770,23 @@ def number_readable(number)
   (numstr.size - 3).step(1, -3) { |i| numstr.insert(i, ",") }
   numstr
 end
+
+def truncate_text_to_approximate_size(s, max_bytes, opts = {})
```

Comment:
Changed `opts` to `options`.

Is `s` not a conventional Ruby/brew abbreviation for "string"? The suffix `_s` is used elsewhere in both brew and main Ruby, so I figured the variable name followed suit.